notebooks/111-yolov5-quantization-migration/README.md

```diff
@@ -0,0 +1,23 @@
+# Migrate quantization from POT API to NNCF
```

Comment:
I would make it consistent: 
Migrate quantization from POT API to NNCF => Migrate quantization from POT API to NNCF API

Reply:
done